src/cmd/package.go

```diff
@@ -195,6 +195,38 @@ var packageRemoveCmd = &cobra.Command{
 	},
 }
 
+var packagePublishCmd = &cobra.Command{
+	Use:   "publish [PACKAGE] [REGISTRY]",
+	Short: "Publish a Zarf package to a remote registry",
+	Example: "  zarf package publish my-package.tar oci://my-registry.com/my-namespace",
+	Args: cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		pkgConfig.PublishOpts.PackagePath = choosePackage(args)
+
+		if !strings.HasPrefix(args[1], "oci://") {
+			message.Fatalf(nil, "Registry must be prefixed with 'oci://'")
+		}
+
+		registry := args[1]
```

Comment:
I would prefer this to be above the if so that the if statement is more readable:
```suggestion
		registry := args[1]
		
		if !strings.HasPrefix(registry, "oci://") {
			message.Fatalf(nil, "Registry must be prefixed with 'oci://'")
		}
```